pycbc/filter/resample.py

```diff
@@ -178,6 +178,34 @@ def resample_to_delta_t(timeseries, delta_t, method='butterworth'):
 _highpass_func = {numpy.dtype('float32'): lal.HighPassREAL4TimeSeries,
                  numpy.dtype('float64'): lal.HighPassREAL8TimeSeries}
 
+def notch_fir(timeseries, f1, f2, order, beta=5.0):
+    """ notch filter the time series using an FIR filtered generated from
+    the ideal response passed through a kaiser window (beta = 5.0)
+
+    The suppression of the notch filter is related to the bandwidth and
+    the number of samples in the filter length. For a few Hz bandwidth,
+    typically severall thousand or ten's of thousands of samples are
```

Comment:
this was more or less what I had in mind .. however, the number of samples is obviously dependent on the sampling rate. not clear if you mean order 10^4 samples at 16384Hz or at (say) 2048Hz.  so, probably better to phrase it as 'a length corresponding to [at least a few seconds] if this is what you mean'